packages/mesh/src/Mesh.js

```diff
@@ -85,6 +87,31 @@ export default class Mesh extends RawMesh
         this.tint = 0xFFFFFF;
 
         this.blendMode = BLEND_MODES.NORMAL;
+
+        /**
+         * TextureMatrix instance for this Mesh, used to track Texture changes
+         *
+         * @member {PIXI.TextureMatrix}
+         * @private
+         */
+        this.uvMatrix = new TextureMatrix(this._texture);
```

Comment:
Also can `uvMatrix` be private?

Reply:
It has clamping changes inside. We might make it readOnly, but its not private.

Reply:
uploadUvMatrix is not readonly.